testsuite/features/step_definitions/command_steps.rb

```diff
@@ -516,7 +516,7 @@
   raise 'File injection failed' unless return_code.zero?
 end
 
-When(/^I copy "([^"]*)" file from "([^"]*)" to "([^"]*)"$/) do |file_path, from_host, to_host|
+When(/^$/) do |file_path, from_host, to_host|
```

Comment:
Did you do this by mistake?

Reply:
Wow, yes, it might be commited by mistake, on my latest push force.